Quick/QuickSpec.m

```diff
@@ -101,7 +101,11 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
         [example run];
     });
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    SEL selector = NSSelectorFromString(example.name.qck_selectorName);
+    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
+                              example.name.qck_selectorName,
+                              example.callsite.file,
+                              (long)example.callsite.line];
+    SEL selector = NSSelectorFromString(selectorName);
```

Comment:
@modocache Regarding
> Is there another way we can fix #299, besides changing the test selector names?

![giphy](https://cloud.githubusercontent.com/assets/352753/7818562/e1d7ec38-03dc-11e5-94a6-af311e77a18a.gif)

So: Definitely Maybe! :)

If this is the only way (or the way we choose), we could use the added line number just for sharedExamples (via testing for `isSharedExample` on `Example`).